src/BenchmarksApps/Websocket/NuGet.config

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should not be necessary. There is one in the root folder with the same sources.